Homework3/Task2Stack/ReverseExtension.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2Stack
+{
+    static class ReverseExtension
+    {
+        public static Stack<T> Reverse<T>(this IStack<T> stack, Stack<T> originalStack)
+        {
+            Array.Reverse(originalStack.MyStack());
```

Comment:
reverse stack parameter here and return reversed result. 
It is ok if passed parameter will be empty after the reversal

Reply:
I don't understand. 
stack is an interface, so I can't reverse anything there.
I need to reverse the array of the Stack<T> class, but I can't access it, since I'm not passing it as a parameter...